ding_collection_reservation.theme.inc

```diff
@@ -58,12 +59,14 @@ function ding_collection_reservation_preprocess_ting_object(&$variables) {
     drupal_add_library('system', 'drupal.ajax');
   }
   elseif ($multiple) {
-    $buttons = &$variables['elements']['ting_primary_object'][0]['ding_entity_buttons'][0];
-    foreach ($buttons as &$button) {
-      // TODO: Get a proper way to find Reservation button.
-      $found = preg_match('/(Res)/', $button['#text']);
-      if (!empty($button['#text']) && $found) {
-        $button['#options']['attributes']['class'][] = 'collection-hidden';
+    if (isset($variables['elements']['ting_primary_object'][0]['ding_entity_buttons'])) {
```

Comment:
We can easily simplify this by removing one level of nesting:
`elseif ($multiple && isset($variables['elements']['ting_primary_object'][0]['ding_entity_buttons'])) {`